### ISSUE TITLE
Crashes immediately after I hit play with MixinBooter

### ISSUE THREAD

**Issue by vainlaind4in** [OpenCubicChunks/CubicWorldGen#127]
Loading MixinBooter 8.0 and above with CubicWorldGen 0.0.85.0 and above makes it crash immediately after hitting play within the Minecraft launcher with exit code 1.
Here's the log:
https://gist.github.com/vainlaind4in/d84dcafa2ed9b292094da9037f45b644

I have been told that it was fixed by [this commit](https://github.com/OpenCubicChunks/CubicWorldGen/commit/d773d7bb62481e29fc87200181ec7051fb83f3fd) but it wasn't implemented into the CurseForge version.

**Comment by RubyPhantasia** [comment]
Yeah, the most recent versions (or close to it) of the .jars can be found in the CC Discord. Could you try using the latest version of CWG (180) from there? The download link will be in the `cc-info-links-dl` channel.
An invite to their Discord is at the bottom of the Cubic Chunks mod page on Curseforge or Modrinth.

**Comment by vainlaind4in** [comment]
Why hasn't it been imported to the main version though?
